fire-hose-management-frontend/src/components/Departments/DepartmentOverlay.jsx

```diff
@@ -1,158 +1,204 @@
 import * as i18nIsoCountries from "i18n-iso-countries";
-import React, { useState } from "react";
+import React, {useState} from "react";
 import Stack from "@mui/material/Stack";
-import { Button, Grid, MenuItem } from "@material-ui/core";
-import { CTextField } from "../shared/input/CTextField";
-import { CSelect } from "../shared/input/CSelect";
-import { ValidatorForm } from "react-material-ui-form-validator";
-import { Overlay } from "../shared/Overlay";
-import { makeStyles } from "@material-ui/core/styles";
+import {Button, Checkbox, FormControlLabel, Grid, MenuItem} from "@material-ui/core";
```

Comment:
Codacy has a fix for the issue: [Replace `Button,·Checkbox,·FormControlLabel,·Grid,·MenuItem` with `⏎··Button,⏎··Checkbox,⏎··FormControlLabel,⏎··Grid,⏎··MenuItem,⏎`](https://app.codacy.com/gh/StitZle/fire-hose-complete/pullRequest?prid=9542260)
```suggestion
import {
```